pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -86,18 +86,18 @@ const (
 	// Prefixes to add to the name of the init containers.
 	// IMPORTANT: Changing these values without changing fluentd collection configuration
 	// will break log collection for init containers.
-	containerPrefix            = "build-step-"
-	unnamedInitContainerPrefix = "build-step-unnamed-"
+	ContainerPrefix            = "build-step-"
+	UnnamedInitContainerPrefix = "build-step-unnamed-"
```

Comment:
instead of exposing these prefixes @vtereso do you think it would be possible for the dashboard logic to be changed such that it doesn't need to hard code these? (i.e. what's the logic where the dashbaord needs this prefix? is it b/c it's looking for the statuses of the actual steps? in that case it could alternatively look for the step names :thinking:?)

Reply:
That's a fair point and I suppose it isn't necessary? Currently, the logging logic uses the container prefixes to determine which containers are `step containers`. I think this was largely due to the confusion on what the nop container was.

Reply:
I think we should different the name of customer defined steps and internal steps (git, source-mkdir, source-copy and nop) in the prefixes of container name.